website/docs/Basic_operations/index.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 1
+---
+
+
+
+# Performing CRUD operations
+
+CRUD (Create, Read, Update, and Delete) operations in FerretDB uses the same protocols and drivers as MongoDB.
+
+## Create operations in FerretDB
+
+The create operation adds a new document to a collection.
+If the collection does not exist, this operation will create it.
+The following methods are available for adding documents to a collection:
+
+[`db.collection.insertOne()`](./create.md#insert-a-single-document), [`db.collection.insertMany()`](./create.md#insert-many-documents-at-once)
+
+## Read operations in FerretDB
+
+The read operation retrieves document records in a collection.
+You can also filter the documents by targeting specific criteria for retrieval.
+The following commands are used to retrieve documents from a collection:
+
+[`db.collection.find()`](./read.md#retrieve-all-documents-in-a-collection), [`db.collection.findOne()`](./read.md#retrieve-a-single-document)
+
+The read operation can also retrieve subdocuments that are nested within a document.
+
+## Update operations in FerretDB
+
+The update operation modifies document records in a collection.
+It changes existing documents in a collection according to the query criteria.
+The following update operations are supported:
+
+[`db.collection.updateOne()`](./update.md#modify-a-single-document), [`db.collection.updateMany()`](./update.md#modify-many-documents), [`db.collection.replaceOne()`](./update.md#replace-a-document)
```

Comment:
We also have `findAndModify` but I guess we don't consider it as a basic CRUD operation. Or should we mention it too?

Reply:
It's not really a basic CRUD operation. Should be covered as we delve deeper into all the functions.